packages/subgraph/src/subgraph.template.yaml

```diff
@@ -110,3 +110,35 @@ dataSources:
         - event: ExercisedCommitment(indexed address,indexed uint256,indexed address,uint256,uint256)
           handler: handleExercisedCommitment
       file: ./src/mapping.ts
+  - kind: ethereum/contract
+    name: CollateralManager
+    network: {{network}}
+    source:
+      address: '{{address_collateral_manager}}'
+      abi: CollateralManager
+      startBlock: {{block_collateral_manager}}
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.5
+      language: wasm/assemblyscript
+      entities:
+        - Collateral
+      abis:
+        - name: CollateralManager
+          file: ./abis/{{network}}_CollateralManager.json
+        - name: IERC20
+          file: ../../packages/contracts/generated/artifacts/@openzeppelin/contracts/token/ERC20/IERC20.sol/IERC20.json
+        - name: IERC721Upgradeable
+          file: ../../packages/contracts/generated/artifacts/@openzeppelin/contracts-upgradeable/token/ERC721/IERC721Upgradeable.sol/IERC721Upgradeable.json
+        - name: IERC1155Upgradeable
+          file: ../../packages/contracts/generated/artifacts/@openzeppelin/contracts-upgradeable/token/ERC1155/IERC1155Upgradeable.sol/IERC1155Upgradeable.json
+      eventHandlers:
```

Comment:
lets also capture `CollateralClaimed` event to update the bid's status to `Liquidated`